app/views/layouts/pageflow/application.html.erb

```diff
@@ -20,7 +20,8 @@
 
   <%= render 'layouts/pageflow/ie_include_tags' %>
 </head>
-<%= content_tag :body, :class => [@body_class, 'load_all_images has_no_mobile_platform non_js'].compact.flatten.join(' ') do %>
+
+<%= content_tag :body, class: "load_all_images has_no_mobile_platform non_js" do %>
```

Comment:
As far as I am concerned yes! You seemed to like the content_tag so I left it in. :)

Will replace with a regular body tag.